core/src/main/scala/cats/instances/future.scala

```diff
@@ -35,6 +35,11 @@ trait FutureInstances extends FutureInstances1 {
 private[instances] sealed trait FutureInstances1 extends FutureInstances2 {
   implicit def catsStdMonoidForFuture[A: Monoid](implicit ec: ExecutionContext): Monoid[Future[A]] =
     new FutureMonoid[A]
+
+  implicit def catsStdParallelForFuture[A](implicit ec: ExecutionContext): Parallel[Future, Future] = {
+    implicit val M: Monad[Future] = cats.instances.future.catsStdInstancesForFuture
+    Parallel.identity[Future]
```

Comment:
what is your thinking on this?

Reply:
I think this is about as valid as the `Monad[Future]`; you get parallel execution using the `Applicative` operators on `Future` anyway, because none of them are lazy, whereas `flatMap` has to be.

Reply:
If you zip two Futures, if I'm not mistaken, it does not complete until the left one completes even if it fails. If the right one fails first, you still have to wait for the left. I can imagine an applicative (with an appropriately loose notion of Exception equality) that fails as soon as either side fails.

Reply:
That's a very interesting idea! Sorry for not responding earlier, I'll give this a shot soon, seems pretty useful :)

Reply:
I created a newtype `FailFastFuture` that behaves in the way you specified, not sure if my implementation is the best, but couldn't think of another way to implement it more succinctly.